### ISSUE TITLE
Feature: possibility to not fire restoreFocus()

### ISSUE THREAD

**Issue by jensdc92** [ncstate-sat/popover#113]
When the popup closes, a method is executed that ensures that the last focused element is focused again. But I show the popup when you drag over an input field. When I use the input field while the popup is still visible, I do not want the previously selected input-field to be focused when the popup closes.

I think it should be possible to disable the restoreFocus() functionality.

Thanks you!

**Comment by willshowell** [comment]
Makes sense. Can you double check that this reproduction is what you're describing?

https://stackblitz.com/edit/sat-popover-issues-pbn8qz?file=app/app.component.html

For now, you might be able to get away with using the `afterClose` event to redirect focus away from the 'previouslyFocusedElement'.

**Comment by jensdc92** [comment]
@willshowell yes this represents the problem.

**Comment by tmonte** [comment]
I'm having the same issue right now. Is there a workaround?

**Comment by willshowell** [comment]
@tmonte sorry I haven't gotten to this yet. If anyone would like to send a PR that makes this optional, I'd really appreciate it!

As a workaround, you can use the `afterOpen` event to set `popover['_previouslyFocusedElement']` to null.


https://stackblitz.com/edit/sat-popover-issues-ajt6ha?file=app%2Fapp.component.html

**Comment by tmonte** [comment]
@willshowell thanks, will use that for now.
